### PR TITLE
Add option to prevent SSL certificate verification 

### PR DESCRIPTION
**Pull request by eduardoj**
This is meant to be used in some development environments, like, for example, an out-of-the-box OBS appliance.

### CODE REVIEW

File: lib/obs_deploy/check_diff.rb [hunk 1]
```diff
@@ -52,11 +63,11 @@ def migrations
     end
 
     def package_url
-      URI("#{@server}/public/build/OBS:Server:Unstable/#{@product}/x86_64/obs-server")
+      "https://build.opensuse.org/public/build/OBS:Server:Unstable/#{@product}/x86_64/obs-server"
```
Comment (danidoni) [inline lib/obs_deploy/check_diff.rb:66]: Should this be hardcoded?

Comment (eduardoj) [reply lib/obs_deploy/check_diff.rb:66]: At least it should be not be the same as the server we are doing the deployment.

Comment (danidoni) [reply lib/obs_deploy/check_diff.rb:66]: Ok, what about extracting that into a properly named variable/constant so we can infer its use by its name? Maybe `REFERENCE_SERVER_URL`?

Comment (eduardoj) [reply lib/obs_deploy/check_diff.rb:66]: > Ok, what about extracting that into a properly named variable/constant so we can infer its use by its name? Maybe `REFERENCE_SERVER_SITE`?

What do you refer exactly by "that"?

Comment (danidoni) [reply lib/obs_deploy/check_diff.rb:66]: Sorry, I'm referring to the `"https://build.opensuse.org"` portion of the url, which is the part you are hardcoding in this PR

**Comment by eduardoj** [comment]
We will not need this: we will retrieve the commit of the installed package with a zypper command in the remote server.

File: lib/obs_deploy/check_diff.rb (hunk 1 above)
Comment (vpereira) [reply lib/obs_deploy/check_diff.rb:66]: Hm, I don't get it too @eduardoj . Aren't you trying to make it flexible to work with different servers?

**Pull request closed without merging**